pkg/ebpf/c/types.h

```diff
@@ -284,6 +284,17 @@ typedef struct path_filter {
     char path[MAX_PATH_PREF_SIZE];
 } path_filter_t;
 
+typedef struct data_filter_key {
+    u32 eventid;
```

Comment:
```suggestion
    u32 event_id;
```
nit: I know that there's still some `eventid`, but it would be nice to standardise them as well.